### PR TITLE
Demo Budget

### LINKED ISSUE #124: create a demo budget for docs contribution
[...] maybe as an alternative you/we could make a demo budget and save it to this repo for testing purposes?

_Originally posted by @j-f1 in https://github.com/actualbudget/docs/pull/123#pullrequestreview-1330588223_
            
Including a small demo budget in the repo will help contributors to get going faster as they can simply import the demo budget and start working with the transactions, accounts, rules, etc that are already in place. 

This will also create some better consistency throughout the documentation with accounts / transactions / payee naming etc.

I would imagine including the budget as a `.zip` backup export from Actual is the best way to deliver?

### PR DESCRIPTION
**Pull request by rich-howell**
Added a demo budget and instructions on how to import it, this should resolve #124

### CODE REVIEW

File: docs/Getting-Started/migration/actual-import.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+---
+title: 'Import an Actual Budget Backup'
+---
+
+While we don't maintain this tool, you can also use https://json-exporter-for-ynab.netlify.app to export your budget from nYNAB into jSON.
```
Comment (j-f1) [inline docs/Getting-Started/migration/actual-import.md:5]: This looks like it’s not relevant to Actual imports, right?

Comment (rich-howell) [reply docs/Getting-Started/migration/actual-import.md]: You're not wrong. Fixed.

**Review by j-f1 — APPROVED** [review]
Thanks!

**Comment by tjex** [comment]
thanks rich 👏

**Pull request merged**